draft-ietf-privacypass-batched-tokens.md

```diff
@@ -459,9 +462,10 @@ struct {
 
 The structure fields are defined as follows:
 
-- GenericBatchTokenRequest's "token_type" is a 2-octet integer. The rest of the
-  structure follows with the TokenRequest based on that type. A TokenRequest
-  with a token type not defined in {{RFC9578}} MAY.
+- GenericBatchTokenRequest's "token_type" is a 2-octet integer. The value
+  represents the token type from the IANA Privacy Pass Token Types Registry
+  ({{AUTHSCHEME, Section 6.2}}). The rest of the structure follows with the
```

Comment:
best to link to the registry directly

```suggestion
  ({{IANA_PRIVACYPASS_TOKEN_TYPES}}). The rest of the structure follows with the
```

In the `informative` section of the header, add the following

```
informative:
  IANA_PRIVACYPASS_TOKEN_TYPES:
    title: IANA Privacy Pass Token Types Registry
    target: https://www.iana.org/assignments/privacy-pass/privacy-pass.xhtml#token-type
```